lib/nexpose/vulnerability.rb

```diff
@@ -112,12 +112,15 @@ def add_bc_to_ssl_cipher_list(source)
     def cleanup_html(source)
       result = source.to_s
       result.gsub!(/<ContainerBlockElement>(.*?)<\/ContainerBlockElement>/m){|m| "#{ $1 }"}
-      result.gsub!(/<Paragraph preformat=\"true\">(.*?)<\/Paragraph>/m){|m| "\nbc. #{ $1 }\n\n"}
+      result.gsub!(/<Paragraph preformat=\"true\">\s*(.*?)<\/Paragraph>/im){|m| "\nbc. #{ $1 }\n\n"}
       result.gsub!(/<Paragraph>(.*?)<\/Paragraph>/m){|m| "#{ $1 }\n"}
       result.gsub!(/<Paragraph>/, '')
       result.gsub!(/<\/Paragraph>/, '')
       result.gsub!(/<UnorderedList>(.*?)<\/UnorderedList>/m){|m| "#{ $1 }"}
-      result.gsub!(/<ListItem>(.*?)<\/ListItem>/m){|m| "#{ $1 }\n"}
+      result.gsub!(/<UnorderedList>|<\/UnorderedList>/){"#{ $1 }"}
+      result.gsub!(/<OrderedList>(.*?)<\/OrderedList>/m){|m| "#{ $1 }"}
+      result.gsub!(/<ListItem>(.*?)<\/ListItem>/m){|m| "* #{ $1 }\n"}
```

Comment:
I think we're still missing something here. We do end up with the tags gone, and the `*` list item present. But we have a group of tabs before the item so it never renders the list

<img width="345" alt="image" src="https://user-images.githubusercontent.com/179134/71897591-5408cc80-3157-11ea-8727-6f670ebf8cb9.png">

<img width="454" alt="image" src="https://user-images.githubusercontent.com/179134/71897622-67b43300-3157-11ea-9fd0-3577dbe31820.png">

Reply:
Maybe we should treat it like the Ordered and Unordered list items and just output the text.

Reply:
Phew, that was complicated but we made it. I think I have this working again - they're bullets and everything!